WordPress/wordpress-playground
# Add log methods, log handlers, and separate log collection

## What is this PR doing?

This PR adds log methods to the `@php-wasm/logger`, log handlers, and separates log collection from the core logger.

## What problem is it solving?

Logging methods will enable us to replace `console...` calls with custom logger calls.
Log handlers allow us to collect logs once and process them in multiple ways.

## How is the problem addressed?

By adding custom logger methods that store messages in the logger and process them using log handlers.

## Testing Instructions

- Checkout this branch
- [Open this blueprint](http://localhost:5400/website-server/?blueprint-url=https://gist.githubusercontent.com/bgrgicak/5fbeb98abec8221192a8b619f852148a/raw/75ff00581b76d9ed3e4844d228dce9b4eda1b163/error-log-blueprint.json)
- Confirm that the PHP error log is visible in the browser console
